ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/node_type_key.schema.yml

```diff
@@ -384,6 +384,62 @@ dynamic_keys:
                     type: bool
                     default: true
 
+          ipvpn_gateway:
+            description: |
+              Node is acting as IP-VPN Gateway for EVPN to MPLS-IP-VPN Interworking | Optional. The BGP peer group used for this is "bgp_peer_groups.ipvpn_gateway_peers".
+              L3 Reachability is required for this to work, the preferred method to establish underlay connectivity is to use core_interfaces.
+            type: dict
+            keys:
+              enabled:
+                type: bool
+                default: false
+              evpn_domain_id:
+                description: Domain ID to assign to EVPN address family for use with D-path. Format <nn>:<nn>.
```

Comment:
I believe there is a default value here and below

Reply:
Fixed